symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/config/BdkExtAppSecurityConfig.java

```diff
@@ -0,0 +1,34 @@
+package com.symphony.bdk.app.spring.config;
+
+import com.symphony.bdk.app.spring.SymphonyBdkAppProperties;
+import com.symphony.bdk.app.spring.properties.CorsProperties;
+
+import lombok.Generated;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import java.util.Map;
+
+/**
+ * Configuration and injection of the security beans within the Spring application context.
+ */
+public class BdkExtAppSecurityConfig {
+
+  @Bean
+  @ConditionalOnProperty("bdk-app.cors")
+  public WebMvcConfigurer corsConfigurer(SymphonyBdkAppProperties properties) {
+    return new WebMvcConfigurer() {
+      // Ignore jacoco check
+      @Generated
```

Comment:
That would be nice to test if `CorsRegistry` is correctly configured from `SymphonyBdkAppProperties` using `Mockito#verify`. But for that you'll need to extend `WebMvcConfigurer` in an inner class: 

```java
public class BdkExtAppSecurityConfig {

  @Bean
  public WebMvcConfigurer corsConfigurer(SymphonyBdkAppProperties properties) {
    return new BdkExtAppWebMvcConfigurer(properties);
  }

  public static class BdkExtAppWebMvcConfigurer implements WebMvcConfigurer {

    private final SymphonyBdkAppProperties properties;

    public BdkExtAppWebMvcConfigurer(SymphonyBdkAppProperties properties) {
      this.properties = properties;
    }

    @Override
    public void addCorsMappings(CorsRegistry registry) {
      // ...
    }
  }
}

```